### PR TITLE
Final PR

### PR DESCRIPTION
**Pull request by kyhjonathan**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -362,14 +358,13 @@ public void updateManager(Scanner scanner, WeekView weekView, MonthView monthVie
...
 
-![UpdateTaskSequenceDiagram.png](images/sequence/UpdateTaskSequenceDiagram.png)
-
 The `updateTask` method is responsible for modifying the details of a task based on the user input.
 
 #### Method Signature
 
 ```
-public static void updateTask(LocalDate date, int taskIndex, String newTaskDescription, Scanner scanner,
+public static void updateTask(LocalDate date, int taskIndex,
```
Comment (Mahesh1772) [inline docs/DeveloperGuide.md:366]: Thanks for the change!

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -571,8 +569,9 @@ The `getFreeTimeSlots` method retrieves all free time slots for a specific date
 
 #### Method Signature
 
-```java
-public List<String> getFreeTimeSlots(List<Task> events, LocalDate currentDate)
+```
```
Comment (Mahesh1772) [inline docs/DeveloperGuide.md:572]: Nice catch!

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -822,17 +829,18 @@ changes to an event task's description, start date, end date, start time, and en
 
 ```java
 private static void updateEventLogging(String newTaskDescription,
-                                       String oldDescription, Event oldEvent, String[] newDatesAndTimes) {
+                                       String oldDescription, Event oldEvent,
```
Comment (Mahesh1772) [inline docs/DeveloperGuide.md:832]: LGTM!

**Review by Mahesh1772 — APPROVED** [review]
LGTM!

**Review by clarencepohh — COMMENTED** [review]
Final PR for the semester!! Nice

**Pull request merged**